pkgs/development/compilers/sagittarius-scheme/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, cmake
+, pkgconfig
+, libffi
+, boehmgc
+, openssl
+, zlib
+, odbc ? true
```

Comment:
It is more common to write this as `odbcSupport`.

Reply:
Done.